packages/tests-proxy/package.js

```diff
@@ -0,0 +1,13 @@
+Package.describe({
+	name: "velocity:test-proxy",
+	summary: "Dynamically created package to expose test files to mirrors",
+	version: "0.0.4",
+	debugOnly: true
+});
+
+Package.onUse(function (api) {
+	api.use("coffeescript", ["client", "server"]);
+	api.add_files("tests/mocha/client/sampleClientTest.js",["client"]);
+	api.add_files("tests/mocha/server/HelpersTest.js",["server"]);
+	api.add_files("tests/mocha/server/sampleServerTest.js",["server"]);
+});
```

Comment:
/packages already contains 143,128 tabs according to RubyMine search.

Unless I misunderstand something (which is quite likely), it would be futile to impose our code standards on external libraries.